adal/src/androidTest/java/com/microsoft/aad/adal/AuthenticationContextTest.java

```diff
@@ -2520,4 +2529,8 @@ void countDownLatch() {
             }
         }
     }
+
+    private Context getContext() {
+        return InstrumentationRegistry.getContext();
```

Comment:
I will remove all these methods, and use the " InstrumentationRegistry.getContext" directly.